data-pipeline/epimodel/gleam/scenario.py

```diff
@@ -291,32 +354,27 @@ def _parse_parameters(self, df: pd.DataFrame):
         is_exception = has_compartment_param & has_exception_fields.all(axis=1)
         is_compartment = has_compartment_param & ~is_exception
         is_multiplier = df["Parameter"].str.contains(" multiplier")
-        is_parameter = ~has_compartment_param & ~is_multiplier
-
-        has_any_exception_fields = has_exception_fields.any(axis=1)
-        bad_parameter = (
-            is_parameter & has_any_exception_fields & (df["Parameter"] != "run dates")
+        is_compartment_scenario = df["Parameter"].str.contains("scenario")
+        is_parameter = (
+            ~has_compartment_param & ~is_multiplier & ~is_compartment_scenario
         )
-        bad_compartment = is_compartment & has_any_exception_fields
-        bad_multiplier = is_multiplier & has_any_exception_fields
-        bad = bad_parameter | bad_compartment | bad_multiplier
-        if bad.any():
-            raise ValueError(
-                "Region and/or dates included with parameters they do not "
-                f"apply to: {df[bad]!r}"
-            )
 
         multipliers = self._prepare_multipliers(df[is_multiplier])
         if multipliers:
             df = df.copy()
             for param, multiplier in multipliers.items():
                 if param not in self.COMPARTMENT_VARIABLES:
                     raise ValueError("Cannot apply multiplier to {param!r}")
-                df.loc[df["Parameter"] == param, "Value"] *= multiplier
```

Comment:
why was this incorrect?

Reply:
It was correct before when all the parameters (loaded from [this csv](data-pipeline/data-dir/inputs/manual/parameters.csv)) were floats. I had to add some strings parameters and after that the column couldn't be parsed as a float. I could try to somehow automatically convert the float string to floats, but since this is the only place, where the values are used arithmetically, it probably doesn't make sense

Reply:
Well, if its like: in the file its an Optional[float] in the column (or whatever causes the value to be always float), I would prefer to retype that wherever the parsing of the file is happening. `pd.read_csv(dtype={"Value": np.float})`

Reply:
But the values are not always floats, they can be strings

Reply:
ah, right, well ok 👍